KStudy.js

```diff
@@ -0,0 +1,509 @@
+				"title": "치과용 콘빔 CT를 이용한 상악 정중과잉치의 3차원 분석",
+				"creators": [
+					{
+						"lastName": "이동호",
+						"creatorType": "author",
+						"fieldMode": 1
+					},
+					{
+						"lastName": "이재서",
+						"creatorType": "author",
+						"fieldMode": 1
+					},
+					{
+						"lastName": "윤숙자",
+						"creatorType": "author",
+						"fieldMode": 1
+					},
+					{
+						"lastName": "강병철",
+						"creatorType": "author",
+						"fieldMode": 1
+					}
+				],
+				"date": "2010",
+				"ISSN": "1229-8212",
+				"abstractNote": "Purpose : This study was performed to analyze the position, pattern of impacted mesiodens, and their relationship to the adjacent teeth using Dental cone-beam CT. Materials and Methods : Sixty-two dental cone-beam CT images with 81 impacted mesiodenses were selected from about 2,298 cone-beam CT images at Chonnam National University Dental Hospital from June 2006 to March 2009. The position, pattern, shape of impacted mesiodenses and their complications were analyzed in cone-beam CT including 3D images. Results : The sex ratio (M : F) was 2.9 : 1. Most of the mesiodenses (87.7%) were located at palatal side to the incisors. 79% of the mesiodenses were conical in shape. 60.5% of the mesiodenses were inverted, 21% normal erupting direction, and 18.5% transverse direction. The complications due to the presence of mesiodenses were none in 43.5%, diastema in 19.4%, tooth displacement in 17.7%, delayed eruption or impaction in 12.9%, tooth rotation in 4.8%, and dentigerous cyst in 1.7%. Conclusions : Dental cone-beam CT images with 3D provided 3-dimensional perception of mesiodens to the neighboring teeth. This results would be helpful for management of the impacted mesiodens.",
+				"issue": "3",
+				"journalAbbreviation": "대한구강악안면방사선학회지 (대한구강악안면방사선학회)",
+				"language": "ko-KR",
+				"libraryCatalog": "KStudy",
+				"pages": "109–114",
+				"publicationTitle": "대한구강악안면방사선학회지 (대한구강악안면방사선학회)",
+				"url": "http://kiss.kstudy.com/public/public2-article.asp?key=50064290",
+				"volume": "40",
+				"attachments": [],
+				"tags": [
+					{
+						"tag": "Cone-Beam Computed Tomograpahy"
+					},
+					{
+						"tag": "Incisor"
+					},
+					{
+						"tag": "Supernumerary"
+					},
+					{
+						"tag": "Tooth"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://kiss.kstudy.com/public/public2-article.asp?key=50789039",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "KOLMOGOROV DISTANCE FOR MULTIVARIATE NORMAL APPROXIMATION",
+				"creators": [
+					{
+						"lastName": "Kim",
+						"creatorType": "author",
+						"fieldMode": 1
+					},
+					{
+						"lastName": "Yoon Tae",
+						"creatorType": "author",
+						"fieldMode": 1
+					},
+					{
+						"lastName": "Park",
+						"creatorType": "author",
+						"fieldMode": 1
+					},
+					{
+						"lastName": "Hyun Suk",
+						"creatorType": "author",
+						"fieldMode": 1
+					}
+				],
```

Comment:
romanized korean names with comma create two last names:

>[ 저자 : Kim, Yoon Tae, Park, Hyun Suk](http://kiss.kstudy.com/public/public2-article.asp?key=50789039)

Korean names are correct imported in "filedMode 1", but it will be nice to have, if korean names can be separated in last and first name like the function "ZU.cleanAuthor" for korean names.

Reply:
Okay, you are right. The RIS data contains wrongly four authors in the `AU` tags. But I still can reuse some of the previous code for scraping it directly in these cases. Stay tuned..